azure-devops/setup.py

```diff
@@ -18,6 +18,7 @@
 # http://pypi.python.org/pypi/setuptools
 
 REQUIRES = [
+    'distro==1.3.0',
```

Comment:
This package has a very scoped usage in the code, instead of bringing/ installing it with overall extension can we wait for it's first time usage before installing it?
we are evaluating something similar at 
 https://github.com/Microsoft/vsts-cli/pull/323 
do share your thoughts